Makefile

```diff
@@ -45,6 +45,8 @@ _scratch:
 	#
 	cp -a ../mynewt-newtmgr/docs _scratch/newtmgr
 	#
+	cp -a ../mynewt-core/docs/os/tutorials/*.rst _scratch/tutorials/
+	#
```

Comment:
Hi Ritika, can you remove this change from the PR? As discussed on slack, we only want to 
copy over files that are converted in mynewt-core. 
Thanks,
-Gavin